enhancements/sig-architecture/43-governance-addon-controller/README.md

```diff
@@ -0,0 +1,118 @@
+# Governance Policy Addon Controller
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [website](https://github.com/open-cluster-management/website/)
+
+## Summary
+
+The goal of this feature is to automate and simplify deployment of the governance policy components
+to managed clusters by creating a controller that leverages the
+[addon framework](https://github.com/open-cluster-management-io/addon-framework/) in order to deploy
+components using the ManagedClusterAddOn kind.
+
+## Motivation
+
+The governance policy framework is currently deployed using individual manifest files deployed
+directly by the user. While this is functional, it does not lend itself to automation,
+customization, scalability, nor hub-centered management that Open Cluster Management demands.
+
+### Goals
+
+- Develop a controller that deploys governance policy components to managed clusters using the
+  ManagedClusterAddOn kind.
+
+### Non-Goals
+
+- None at this time.
+
+## Proposal
+
+### User Stories
+
+1. As a k8s admin, I want to deploy, uninstall, and customize the governance policy framework on my
+   managed clusters.
+
+### Architecture
+
+The governance policy addon controller manages installations of policy addons on managed clusters by
+using
+[ManifestWorks](https://github.com/open-cluster-management-io/api/blob/main/docs/manifestwork.md).
+The addons can be enabled, disabled, and configured via their `ManagedClusterAddOn` resources.
+
+(For more information on the addon framework, see the
+[addon-framework enhancement/design](https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/8-addon-framework).)
+
+The addons managed by this controller are:
+
+- The "config-policy-controller" consisting of the
+  [Configuration Policy Controller](https://github.com/open-cluster-management-io/config-policy-controller).
```

Comment:
so the policy propagator will not be covered in the addon, is it? currently addon-framework doesnt automate the installation of hub-side components such as controllers/server for us, @qiujian16 what do you think about supporting hub-side addon installation in the addon-framework?

Reply:
this sounds like some thing an operator should do :) but I am fine with that. I think that would need another interface in the addon framework to define the hub component maybe? And we also need to think about how user can set the parameter for these components.